src/lucky/renderable.cr

```diff
@@ -34,18 +35,28 @@ module Lucky::Renderable
   end
 
   private def handle_response(response : Lucky::Response)
+    log_response(response)
     response.print
   end
 
   private def handle_response(_response : T) forall T
-    {% raise <<-ERROR
+    {%
+      raise <<-ERROR
 
-     #{@type} returned #{T}, but it must return a Lucky::Response.
+      #{@type} returned #{T}, but it must return a Lucky::Response.
 
-     Try this...
-       ▸ Make sure to use a method like `render`, `redirect`, or `json` at the end of your action.
-       ▸ If you are using a conditional, make sure all branches return a Lucky::Response.
-    ERROR %}
+      Try this...
+        ▸ Make sure to use a method like `render`, `redirect`, or `json` at the end of your action.
+        ▸ If you are using a conditional, make sure all branches return a Lucky::Response.
+      ERROR
+    %}
+  end
+
+  private def log_response(response)
+    debug_message = response.debug_message
+    if !debug_message.nil?
```

Comment:
You can do `if debug_message` to keep this a bit more concise, or `try` might make it even cleaner:

```crystal
response.debug_message.try do |message|
  context.add_debug_message(message)
end
```